context/app/static/js/components/Routes.jsx

```diff
@@ -65,6 +66,14 @@ function Routes(props) {
       </Container>
     );
   }
+
+  if ('markdown' in flaskData) {
```

Comment:
```suggestion
  if ('markdown' in flaskData && urlPath.startsWith('/docs')) {
```

Reply:
Right now, it could also be `VERSION`... and I feel like at some point there will be another addition at the top level, and tracking down why it doesn't render will be annoying.